RandLAPACK/drivers/rl_bqrrp.hh

```diff
@@ -55,17 +60,15 @@ class BQRRP : public BQRRPalg<T, RNG> {
 
         BQRRP(
             bool time_subroutines,
-            T ep,
             int64_t b_sz
         ) {
             timing          = time_subroutines;
-            eps             = ep;
             tol             = std::numeric_limits<T>::epsilon();
             block_size      = b_sz;
             internal_nb     = b_sz;
-            qrcp_wide       = "luqr";
-            qr_tall         = "geqrf";
-            apply_trans_q   = "ormqr";
+            qrcp_wide       = BQRRP<T, RNG>::SubroutineType::qrcp_wide_subroutines::luqr;
+            qr_tall         = BQRRP<T, RNG>::SubroutineType::qr_tall_subroutines::geqrf;
+            apply_trans_q   = BQRRP<T, RNG>::SubroutineType::apply_trans_q_subroutines::ormqr;
```

Comment:
You don't need to fully qualify the scopes here. With your current implementation it should suffice to have lines like the following.
```c++
qrcp_wide = SubroutineType::qrcp_wide_subroutines::luqr;
```
With my suggested implementation it should suffice to have lines like this:
```c++
qrcp_wide = Options::QRCPWide::LUQR
```

Reply:
Thought so as well, but i get compiler errors if i don't use the full scope.

Reply:
Hmm. Well, you can throw a using statement at the top of the class definition, like I do here in RandBLAS:
https://github.com/BallisticLA/RandBLAS/blob/dea0ae09e84ea20a1ffec757cf09e33b32ac1ab3/RandBLAS/dense_skops.hh#L337-L347

Reply:
Done